### DISCUSSION TITLE
Federated collections (share collection between multiple installations)

### DISCUSSION THREAD

**Discussion by richardhj** [statamic/cms#8890] [General]
I think it would be super cool (and handy for my use case) to share the same collection (of events, news, ...) on multiple installations.

On the server side, one can achieve this by symlinking the content folder (or subfolders). When the installations are on different servers, one might utilize the API to (dynamically) hydrate the collection.

Is there someone who did this already?

**Comment by ryanmitchell** [comment]
You could use Runway for that collection, make it database driven and then you can access it from multiple installs.

**Comment by richardhj** [comment]
I could use runway but want to fetch from the other system's API instead. 
Is there a way to extend repositories in a way, that one colleciton pulls data from API instead of stache?

**So that I can use**
```{{ collection from="local_events|remote_events" }}``` where `local_events` is my ordinary collection and `remote_events` is what I define in the repository?
